torchgeo/datasets/__init__.py

```diff
@@ -92,6 +93,7 @@
     "RESISC45",
     "SEN12MS",
     "So2Sat",
+    "Spacenet1",
```

Comment:
Since this is a `GeoDataset`, this should go a few lines up directly above `# VisionDataset`

Reply:
Fixed